### PR TITLE
Add supporting to use Amazon S3 file for query execution

### PR DESCRIPTION
**Pull request by daichi-hirose**
* [Enhancement]  Add supporting to use Amazon S3 file for query execution

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ plugins {
 }
 
 group = 'pro.civitaspo'
-version = '0.1.3'
+version = '0.1.4'
```
Comment (civitaspo) [inline build.gradle]: Could you revert this? Bumping up version is responsible for me.

File: example/example.dig [hunk 2]
```diff
@@ -4,7 +4,7 @@ _export:
       - file://${repos}
       # - https://jitpack.io
     dependencies:
-      - pro.civitaspo:digdag-operator-athena:0.1.3
+      - pro.civitaspo:digdag-operator-athena:0.1.4
```
Comment (civitaspo) [inline example/example.dig]: ditto

File: build.gradle (hunk 1 above)
Comment (daichi-hirose) [reply build.gradle]: fixed

File: example/example.dig (hunk 2 above)
Comment (daichi-hirose) [reply example/example.dig]: fixed

**Pull request merged**